### PR TITLE
Add default kick msg component

### PR DESCRIPTION
**Pull request by Machine-Maker**
If someone knows about a patch this can go it, pls post it.

### CODE REVIEW

File: patches/server/0843-Add-default-kick-msg-component.patch [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 4ff4143f3a7cd89ef92f4b8882fa3e5addfe0f06..afa6125909767c23a6ac803116058cb8e68173eb 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -504,6 +504,10 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+ 
+     // Paper start
+     @Override
++    public void kick() {
++        this.kick(net.kyori.adventure.text.Component.translatable("multiplayer.disconnect.kicked"));
```
Comment (montlikadani) [inline patches/server/0843-Add-default-kick-msg-component.patch]: Would be good to cache this component

**Review by Owen1212055 — APPROVED** [review]
Works fine, a nice little convenient method.
image

**Comment by Owen1212055** [comment]
> If someone knows about a patch this can go it, pls post it.

And what about putting this in the adventure patch? It's just a little method addition.

**Comment by Machine-Maker** [comment]
Ok, moved to the Adventure patches in -Server and -API

**Review by kashike — APPROVED** [review]

**Pull request merged**